### DISCUSSION TITLE
how to use the api? Could someone help me plz!

### DISCUSSION THREAD

**Discussion by AswarS** [apache/superset#19652] [Q&A / Help]
i want to use the api to get and create a database, dataset and chart, but i don't understand the parameter provided by the offical document.((http://127.0.0.1:5000/swagger/v1))
for example, i want to get the charts information, but what does the columns means and what value should it take?
The following are the parameters required by the api 
{
  "columns": [
    "string"
  ],
  "filters": [
    {
      "col": "string",
      "opr": "string",
      "value": 0
    }
  ],
  "keys": [
    "list_columns"
  ],
  "order_column": "string",
  "order_direction": "asc",
  "page": 0,
  "page_size": 0
}

**Comment by srinify** [comment]
Hey Aswar! Moving to Discussions since this isn't a bug (we're trying to keep Issues focused on Bugs and SIPs).

You may also find this introductory event useful around the Superset API: https://www.youtube.com/watch?v=vFTI087HYNE
